src/screens/timeStamping/EventEdition/index.tsx

```diff
@@ -99,7 +99,7 @@ const EventEdition = ({ route, navigation }: EventEditionProps) => {
     kmDuringEvent: false,
   });
   const [internalHourOptions, setInternalHourOptions] = useState<InternalHourOptionsType[]>([]);
-  const [subHeader, setSubHeader] = useState<subHeaderType>({ text: '', bgColor: '', textColor: '' });
+  const [subHeader, setSubHeader] = useState<subHeaderType>({ text: '', bgColor: WHITE, textColor: WHITE });
```

Comment:
J'ai changé les valeurs d'initialistion car on ne peut pas initialiser des couleurs a '', un warning `Failed prop type: Invalid prop `backgroundColor` supplied to `Text`` s'affichait